### PR TITLE
Armour Check

### PR DESCRIPTION
**Pull request by yagizher**

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -39,14 +39,22 @@ function setUniform(uniform, playerPed)
...
 	end)
 end
 
+function armourcheck()
+    if GetPedArmour(GetPlayerPed(-1)) > 99 then
+        ESX.ShowNotification(_U('already_armour'))
+    else
+        SetPedArmour(GetPlayerPed(-1), 100)
+    end
+end)
```
Comment (XvenDeR) [inline client/main.lua:56]: dont need ) at the end

**Comment by yagizher** [comment]
Function or the trigger?

**Comment by yagizher** [comment]
updated

**Comment by FBFezz** [comment]
change cin client the label to armour_already as you have it backwards

**Comment by FBFezz** [comment]
You just changed it again, you need the local and the main to be the sam so armour_already for both

**Comment by yagizher** [comment]
ooh lol my fault

File: locales/en.lua [hunk 2]
```diff
@@ -14,6 +14,7 @@ Locales['en'] = {
   ['put_weapon'] = 'store weapon in armory',
   ['buy_weapons'] = 'buy weapons',
   ['armory'] = 'armory',
+  ['already_armour'] = 'You already have an ~y~Armour!~s~.',
```
Comment (XvenDeR) [inline locales/en.lua:17]: change to ['armour_already'] = 'You already have an ~y~Armour!~s~.',

File: client/main.lua [hunk 3]
```diff
@@ -39,14 +39,23 @@ function setUniform(uniform, playerPed)
...
 		else
 			ESX.ShowNotification(_U('no_outfit'))
 		end
 	end)
 end
 
+function armourcheck()
+	local player = GetPlayerPed(-1)
+    if GetPedArmour(player) > 99 then
+        ESX.ShowNotification(_U('already_armour'))
```
Comment (XvenDeR) [inline client/main.lua:53]: ESX.ShowNotification(_U('armour_already'))

**Comment by yagizher** [comment]
Done

**Comment by FBFezz** [comment]
Still dont get a notification for some reason even tho that would normally be right

**Pull request closed without merging**